refresh.yml

```diff
@@ -0,0 +1,20 @@
+app_root: .
+binary_name: refresh-build
+build_delay: 200ns
+build_flags: []
+build_path: /var/folders/00/nfy691j564dchmft4_3cdhpr0000gn/T/
```

Comment:
@naman-dave is this build_path your system's own?

Reply:
@paraswaykole Nope. This file get created during `refresh init` so all of this is default configs. but we can give it a build path.